packages/bff/src/services/authorizationService.ts

```diff
@@ -0,0 +1,37 @@
+import { authorizationApi } from "pagopa-interop-api-clients";
+import { AuthorizationProcessClient } from "../providers/clientProvider.js";
+
+export const getAllClients = async (
```

Comment:
In https://github.com/pagopa/interop-be-monorepo/pull/753 @paologaleotti is putting clients-related stuff into `clientService.ts`, I would use the same approach and move these into a `clientsService.ts` file